test/sdk-execution-tests/sdk_execution_tests.py

```diff
@@ -125,35 +126,9 @@ def run(test_case: TestCase) -> Tuple[str, client.client.RunPipelineResult]:
     return run_url, run_result
 
 
-def get_run_artifacts(run_id: str):
-    mlmd_connection_config = metadata_store_pb2.MetadataStoreClientConfig(
-        host=METADATA_HOST,
-        port=METADATA_PORT,
-    )
-    mlmd_store = metadata_store.MetadataStore(mlmd_connection_config)
-    contexts = mlmd_store.get_contexts(list_options=ListOptions(filter_query=f"name = '{run_id}'"))
-    if len(contexts) != 1:
-        print("ERROR: Unable to find pipelinerun context in MLMD", file=sys.stderr)
-        return []
-
-    context = contexts[0]
-    return [a for a in mlmd_store.get_artifacts_by_context(context.id)]
-
-
 def cleanup_run_resources(run_id: str):
     print(f"Cleaning up resources for run {run_id}")
 
-    artifacts = get_run_artifacts(run_id)
-    print(f"Found {len(artifacts)} artifacts for run {run_id}")
-    # Clean up any Artifacts from object store
-    for artifact in artifacts:
-        try:
-            object_key = artifact.uri.removeprefix(f"minio://{BUCKET_NAME}")
-            print(f"Deleting artifact {object_key} for run {run_id}")
-            minio_client.remove_object(BUCKET_NAME, object_key)
-        except S3Error as err:
-            print(f"MinIO error: {err} for run {run_id}")
-
```

Comment:
Why was this removed? Is it no longer needed?

Reply:
mlmd doesn't support >= 5 for proto buf: https://github.com/google/ml-metadata/blob/master/setup.py#L169 
so we can no longer use this client along with tooling that's utilizing kfp. The `Free up space in /dev/root` is enough today to avoid the flakiness we experienced before, and this change [here](https://github.com/kubeflow/pipelines/pull/11965/files#diff-ef3f59e6590d783bfe41c212d12be7c108bb388d6b085f99f7103a8d8d9a0017) will introduce more space cleanup. Still, once this is merged I will add a follow up task to add a replacement for this logic. Wdyt?